scripts/checks.py

```diff
@@ -169,7 +169,28 @@ def check_recombine_ics(directory=None, startsec=None, n_secs=None, required_siz
     return error, files_in_dir
 
 
-def get_files_and_sizes(obsID, mode):
+def get_files_and_sizes(obsID, mode, mintime=0, maxtime=2000000000):
+    """
+    Get files and sizes from the MWA metadata server and check that they're all the same size
+
+    Parameters:
+    -----------
+    obsID: int
+        The MWA observation ID
+    mode: str
+        The typ of file from 'raw', 'tar_ics' and 'ics'
+    mintime: int
+        The minimum GPS time of observations to check (inclusive, >=)  Default: 0
+    maxtime: int
+        The maximum GPS time of observations to check (exculsive, <)  Default: 2000000000
```

Comment:
How come you've made this exclusive when every other example of grabbing data is inclusive?

Reply:
Because the metadata call is exclusive. Also, some of our internal functions are exclusive (which can be confusing hence documentation)